README.md

```diff
@@ -189,19 +197,19 @@ You can download the Node.JS bindings using `npm`:
 npm install deepspeech
 ```
 
-Alternatively, if you're using Linux and have a supported NVIDIA GPU (See the release notes to find which GPU's are supported.), you can install the GPU specific package as follows:
+Alternatively, if you're using Linux and have a supported NVIDIA GPU, you can install the GPU specific package as follows:
 
 ```bash
 npm install deepspeech-gpu
 ```
 
-See [client.js](native_client/javascript/client.js) for an example of how to use the bindings.
+See the [release notes](https://github.com/mozilla/DeepSpeech/releases) to find which GPUs are supported. Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
-Please ensure you have the required [CUDA dependency](#cuda-dependency).
+See [client.js](native_client/javascript/client.js) for an example of how to use the bindings.
```

Comment:
We now have a `examples/nodejs_wav` to link to, which is more straightforward than the client.js file.

Reply:
see 19d9210051d38d59b712ba9fd137316afbc83866